style.css

```diff
@@ -381,6 +381,39 @@ body {
 .shadow-cont{
     box-shadow: 5px 5px 5px 5px var(--off-white);
 }
+
+/* ------------avatar-feature-------------- */
+.avatar-container{
+    padding:1rem;
+    display:flex;
+    flex-direction:column;
```

Comment:
You can give a class name as flex-col and have the code in that class and just use that class everywhere.

Reply:
I will use it. Thank you